llmlite/llms/model.py

```diff
@@ -73,20 +73,24 @@ def load_with_vllm(
         }
         return cls(model_name_or_path, **config_args)
 
-    def completion(self, messages: List[ChatMessage], **kwargs) -> Optional[str]:
+    def completion(self, messages: List[List[ChatMessage]], **kwargs) -> Optional[Union[str, List[str]]]:
```

Comment:
List should be `Union` type

Reply:
I have processed the messages to List[List[ChatMessage]] in the start.

Reply:
got it.